src/OpenStudy/study_openinterface.jl

```diff
@@ -122,6 +122,17 @@ end
 # TODO: preprocess date vectors
 # TODO: informs about empty slots (beyond classes)
 
+function _merge_collections!(data::Data)
+    raw = _raw(data)
+    #PSRSerie <= PSRTransformer
+    if haskey(raw, "PSRSerie") && haskey(raw, "PSRTransformer") 
+        append!(raw["PSRSerie"], raw["PSRTransformer"])
+    elseif haskey(raw, "PSRTransformer") 
+        raw["PSRSerie"] = raw["PSRTransformer"]
+    end
+    nothing
+end
```

Comment:
@Thiago-NovaesB I would do this a little different, the function could be 
````julia
function _merge_collections!(data:::Data kwargs...)
    if add_transformers_to_series
        _merge_psr_transformer_and_psr_serie(data)
    end
    return nothing
end
```
and then the merge itself would go intro this function